### PR TITLE
chore: add commit hook for message format

### PR DESCRIPTION
**Pull request by Lauraducky**

### CODE REVIEW

**Comment by jimcullenaus** [comment]
What is "chore". I can't find mention of it in their doco

File: commitlint.config.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+module.exports = {
```
Comment (Chromadream) [inline commitlint.config.js]: if we're importing `@commitlint/config-conventional`, why are we pasting the rules manually here?

Comment (Lauraducky) [reply commitlint.config.js]: Whoops, I guess I messed up the config a bit. Fixed now

**Comment by Lauraducky** [comment]
> What is "chore". I can't find mention of it in their doco

"chore" is: Other changes that don't modify src or test files

**Pull request merged**